services/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -34,7 +34,11 @@
             <cm:property name="definitions.refresh.interval" value="10000"/>
             <cm:property name="properties.refresh.interval" value="10000"/>
             <cm:property name="segment.refresh.interval" value="1000"/>
+            <cm:property name="segment.max.retries.update.profile.segment" value="5"/>
+            <cm:property name="segment.retry.update.segment.seconds.delay" value="1"/>
```

Comment:
This one is not listed in the configurable values, is it normal ?

Reply:
right, added it for "segment.retry.update.segment.seconds.delay" as well